### PR TITLE
fixed several grammatical errors and inconsistent spelling throughout…

### PR DESCRIPTION
**Pull request by judyz01**
… site

I fixed several grammatical errors throughout the site

## Description
Made several passages more concise and easy to read. Kept brand image consistent by changing VetsWhoCode to #VetsWhoCode. Several mentions of Javascript or javascript were changed to JavaScript. Multiple instances of "students" were changed to "mentees" to maintain consistency. 

## Motivation and Context
Hacktoberfest!

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -71,52 +71,47 @@ function About() {
...
                 <p className="story">
-                  Vets Who Code is a reintegration solution for veterans that believes in the
-                  principle of &quot;To Teach a Man To Fish&quot; to better prepare early stage
-                  transitioning veterans for returning to the workforce. Many veterans are not
-                  homeless, but are un- and under-employed. Some have a deep desire to be
-                  independent business owners. Their skill sets are desperately needed to enhance
-                  local economies and to drive innovation. Vets Who Code serves as a launch pad for
+                  Vets Who Code is a reintegration solution for veterans who believe in the
+                  principle of &quot;To Teach a Man To Fish&quot; to better prepare early-stage
+                  transitioning veterans who are returning to the workforce. Many veterans are are un- and under-employed.
```
Comment (gixxerblade) [inline src/pages/about.js:85]: "are are" is not correct grammar

File: src/pages/about.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ function About() {
               <h1 className="story-title">About Us</h1>
               <p>
                 <i>
-                  #VetsWhoCode Is a Non-ForProfit where veterans teach veterans how to program and
+                  #VetsWhoCode is a Non-ForProfit where veterans teach veterans how to program and
```
Comment (gixxerblade) [inline src/pages/about.js:41]: Change Non-ForProfit to Non-For Profit

File: src/pages/about.js [hunk 3]
```diff
@@ -71,52 +71,47 @@ function About() {
...
 
                 <h2>What We Do</h2>
                 <p className="story">
                   At Vets Who Code, we take a small cohort of veterans and spouses twice a year and
-                  over the course of sixteen weeks train them in programming with a deep focus on
-                  Javascript, the language of the web. We do this all remotely using the best tools
-                  and resources the tech community has to offer. We do this at zero cost to the
-                  troop beyond them contributing to one of our teams and helping each other become
-                  better programmers.
+                  train them in programming with a deep focus on JavaScript, over the course of sixteen weeks.
```
Comment (gixxerblade) [inline src/pages/about.js:94]: Re-add after `JavaScript", "the language of the web."

File: src/pages/about.js [hunk 4]
```diff
@@ -71,52 +71,47 @@ function About() {
...
                 <p className="story">
-                  How we accomplish this mission is through a process of <i>Crawl, Walk, Run</i>
-                  where as we teach them programming we build upon each lesson in deeper dives so
-                  that they become better programmers through each iteration. All this while being
-                  lead by instructors who are also veterans, and programmers, and alumni of the
-                  program. That way there are no excuses, the person teaching you the craft has been
+                  How we accomplish this mission is through a process of <i>Crawl, Walk, Run </i>
+                  where we build upon each lesson in deeper dives so that veterans become better programmers through each iteration. 
+                  Instructors leading the program are also veterans, programmers, and alumni of the
+                  program. There are no excuses, the instrcutor has been
```
Comment (gixxerblade) [inline src/pages/about.js:104]: Correct "instrcutor " to "instructor"

File: src/pages/about.js [hunk 5]
```diff
@@ -71,52 +71,47 @@ function About() {
...
                 </p>
 
                 <h2>WE NEED YOUR HELP</h2>
                 <p className="story-last">
                   As our country is going through this unprecedented time with COVID-19, people are
-                  wanting to learn the skills of today and tomorrow to future-proof themselves so
-                  that they can provide for their families. Furthermore due to the nature of
-                  for-profit code schools many are just now learning how to do the work remotely,
-                  which we have been doing for years. As a result we are having an unprecedented
+                  wanting to learn the skills of today and tomorrow to future-proof themselves in order to provide for their families.
```
Comment (gixxerblade) [inline src/pages/about.js:112]: Remove "in order" add a comma after "themselves"

File: src/pages/about.js [hunk 6]
```diff
@@ -71,52 +71,47 @@ function About() {
...
 
                 <h2>WE NEED YOUR HELP</h2>
                 <p className="story-last">
                   As our country is going through this unprecedented time with COVID-19, people are
-                  wanting to learn the skills of today and tomorrow to future-proof themselves so
-                  that they can provide for their families. Furthermore due to the nature of
-                  for-profit code schools many are just now learning how to do the work remotely,
-                  which we have been doing for years. As a result we are having an unprecedented
+                  wanting to learn the skills of today and tomorrow to future-proof themselves in order to provide for their families. 
+                  Furthermore, due to the nature of for-profit coding schools, many are now learning how to do the work remotely,
```
Comment (gixxerblade) [inline src/pages/about.js:113]: Remove "Furthermore" and start the sentence with "Due"

Comment (gixxerblade) [reply src/pages/about.js:113]: Change "many are now learning how to do the work remotely," to "many are now learning how to work remotely,"

File: src/pages/about.js [hunk 7]
```diff
@@ -71,52 +71,47 @@ function About() {
...
+                  of which we have been doing for years. As a result, we are having an unprecedented
```
Comment (gixxerblade) [inline src/pages/about.js:114]: Change "of which we have been doing for years." to "which we have been doing for years."

File: src/pages/mentor.js [hunk 8]
```diff
@@ -164,16 +162,16 @@ function Mentor() {
...
                     <h4 className="small-top-pad">Time Commitment</h4>
                     <p>
-                      After being formally assigned a mentee we ask mentors to set up a time for an
-                      initial chat with their mentee to discuss their needs and generally get to
+                      After being formally assigned a mentee, we ask mentors to set up an
+                      initial chat with their mentee to discuss their needs and to get to
                       know one another. After this initial meeting, we ask that our mentors check in
-                      with their mentee at least once a week to inquire about their progress and if
-                      the students have any issues.
+                      with their mentee at least once a week to inquire about their progresses and if
```
Comment (gixxerblade) [inline src/pages/mentor.js:168]: Change back to "progress"

File: src/pages/mentor.js [hunk 9]
```diff
@@ -164,16 +162,16 @@ function Mentor() {
...
+                      the mentees have any issues.
                     </p>
-                    VetsWhoCode is a virtual organization and Slack is the primary vehicle for
-                    communication and activity. We ask mentors to be available in Slack to students
-                    and to use slack as a way to check in with mentees on a weekly basis. Mentors
-                    and students may also use email or video chat to communicate with one another.
+                    #VetsWhoCode is a virtual organization in which Slack is the primary vehicle for
```
Comment (gixxerblade) [inline src/pages/mentor.js:171]: Change "in which" to "where"

File: src/pages/syllabus.js [hunk 10]
```diff
@@ -10,25 +10,25 @@ export default class Mentor extends Component {
...
         body:
-          'Block two is all about computer science. As real world trends prove to us that web apps are growing in complexity we need to understand the tools to be able to create efficiencies and faster processes. This begins with the computer science elements learned through JavaScript. ',
+          'Block 2 is all about computer science. As real world trends prove to us that web apps are growing in complexity, we need to understand the tools to be able to create efficiencies and faster processes. This begins with the computer science elements learned through JavaScript. ',
       },
       {
         title: 'Block 3: React & JAMstack',
         id: 'b-3',
         body:
-          'Block three introduces the JAMstack. Utilizing Gatsby which relies upon the best tech, such as React and GraphQL, we can make sure our students are getting the best and most sought after skills in our industry today.',
+          'Block 3 introduces the JAMstack. Utilizing Gatsby, which relies upon the best technology, such as React and GraphQL, we can make sure our students are getting the best and most sought-after skills in our industry today.',
```
Comment (gixxerblade) [inline src/pages/syllabus.js:25]: Change "we can make sure our students" to "we can ensure our students"

File: src/pages/syllabus.js [hunk 11]
```diff
@@ -45,12 +45,11 @@ export default class Mentor extends Component {
                 <div className="faq-short-brief">
                   <h2>#VetsWhoCode Syllabus</h2>
                   <p>
-                    Our curriculum is designed using the Agile Methodology with four-day sprints
+                    Our curriculum is designed using the Agile Methodology with 4 day sprints
```
Comment (gixxerblade) [inline src/pages/syllabus.js:48]: Change "4 day sprints" back to "four-day sprints"

File: src/pages/syllabus.js [hunk 12]
```diff
@@ -45,12 +45,11 @@ export default class Mentor extends Component {
...
                     dedicated to the subject. We do it in this manner based on the practice of how
-                    veterans &quot;Train How You Fight&quot; in the military, so they should train
-                    how they would work in a real software company. The primary language we will
-                    focus on is JavaScript, due to its popularity in the job market, ease of
-                    transition between Computer Science topics and real world application, and the
+                    veterans &quot;Train How You Fight&quot; in the military, the practice of which is now applied towards working in a software company. 
+                    The primary language we will focus on is JavaScript, due to its popularity in the job market, ease of
+                    transition between computer science topics and real world application, and the
```
Comment (gixxerblade) [inline src/pages/syllabus.js:52]: Add a comma after "topics"

**Comment by judyz01** [comment]
Hey @gixxerblade, I made the appropriate changes, let me know if there's anything else that needs adjusting.

File: src/pages/about.js [hunk 13]
```diff
@@ -71,52 +71,47 @@ function About() {
...
 
                 <h2>WE NEED YOUR HELP</h2>
                 <p className="story-last">
                   As our country is going through this unprecedented time with COVID-19, people are
-                  wanting to learn the skills of today and tomorrow to future-proof themselves so
-                  that they can provide for their families. Furthermore due to the nature of
-                  for-profit code schools many are just now learning how to do the work remotely,
-                  which we have been doing for years. As a result we are having an unprecedented
+                  wanting to learn the skills of today and tomorrow to future-proof themselves, in order to provide for their families. 
+                  Due to the nature of for-profit coding schools, many are now learning how to work remotely,
```
Comment (gixxerblade) [inline src/pages/about.js:113]: Remove "how"  in "many are now learning how to work remotely,"

**Review by gixxerblade — CHANGES_REQUESTED** [review]

**Comment by judyz01** [comment]
Done! @gixxerblade

**Review by gixxerblade — APPROVED** [review]

**Comment by gixxerblade** [comment]
Thank you very much! Good luck with the rest of Hacktoberfest!

**Pull request merged**